src/lib/bitcoin/BitcoinRequestParserMixin.js

```diff
@@ -0,0 +1,234 @@
+/* global Nimiq */
+/* global Errors */
+/* global BitcoinJS */
+/* global BitcoinUtils */
+
+/**
+ * @template {{}} T
+ * @typedef {new (...args: any[]) => T} Constructor;
+ */
+
+// eslint-disable-next-line valid-jsdoc
+/**
+ * A mixin to add Bitcoin-related request parsers to a RequestParser class.
+ *
+ * @template {Constructor<RequestParser>} TBase
+ * @param {TBase} clazz
+ * @returns {typeof Clazz}
+ */
+function BitcoinRequestParserMixin(clazz) { // eslint-disable-line no-unused-vars
```

Comment:
This file contains the Bitcoin-related request parsing methods that you already reviewed in the `bitcoin` branch PR. Only minor things changed in the actual methods, which I tried to mark with these review comments.